src/Plugin/WebformHandler/DigitalPostWebformHandler.php

```diff
@@ -314,8 +314,9 @@ public function postSave(WebformSubmissionInterface $webform_submission, $update
       $element = $this->webform->getElement($key);
 
       $elements[] = [
+        'label' => $this->webform->label(),
         'name' => $element['#title'],
-        'value' => $value,
+        'value' => isset($element['#return_value']) ? $element['#return_value'] : $value,
```

Comment:
Write this as `$element['#return_value'] ?? $value`.

Reply:
Changed